manifest.json

```diff
@@ -28,7 +46,7 @@
   "proxy": {
     "whitelist": [
       {
-        "url": "https://www.wrike.com/.*",
+        "url": "https://.*",
```

Comment:
For security, this seems too open. Can we restrict it to any `*.wrike.com` instead?

Reply:
At the moment, it's not possible to make this more secure because we're not certain what the value might be. Wrike.com might not always be available. When [this story](https://app.shortcut.com/deskpro/story/160898/custom-apps-string-default-values-aren-t-working) is done, we'll be able to fix it as `__instance_url__/.*`.